README.rst

```diff
@@ -448,6 +448,62 @@ SERVERS
     to make multihost prototype support easier to verify as many current
     tests were developed with Restraint.
 
+Plugin options
+--------------
+
+.. note::
```

Comment:
I understand it's a kind of warning, but as a new user coming to the section I don't know the context and trying to digest the exception is hard without knowing what is it about, thus I still have to read first what's below...

Reply:
Fixed in e79f6f2b.